.github/workflows/llm_integration.yml

```diff
@@ -52,10 +52,18 @@ jobs:
         run: |
           docker pull deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG
           mkdir logs
+      - name: Test bloom-7b
+        working-directory: tests/integration
+        run: |
+          sudo python3 deepspeed/prepare.py bloom-7b1
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          serve -m test=file:///opt/ml/model/test/
+          python3 deepspeed/client.py bloom-7b1
+          docker rm -f $(docker ps -aq)
       - name: Test GPTJ-6B
         working-directory: tests/integration
         run: |
-          python3 deepspeed/prepare.py gpt-j-6b
+          sudo python3 deepspeed/prepare.py gpt-j-6b
```

Comment:
Why we need sudo here?

Reply:
got permission error when removing files